app/logic/util/util.ts

```diff
@@ -76,6 +76,16 @@ export function fileExtensionForMIMEType(mimetype: string) {
   return ".ext";
 }
 
+export async function throttleConnectionsPerSecond(nextConnectionTime: number[]) {
```

Comment:
This parameter **really** needs documentation. Who populates it? (This function, not the caller, just pass it through) How to init it (array length = number of allowed calls per second, and init all values with 0)